jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2675,6 +2675,36 @@ A job can have the keys `requires`, `name`, `context`, `type`, and `filters`.
 A job name that exists in your `config.yml`.
 
 '''
+
+[#override-with]
+====== *`override-with`*
+
+The `override-with` key is used to override the job configuration with a job in the referenced orb.
+
+[.table.table-striped]
+[cols=4*, options="header", stripes=even]
+|===
+| Key | Required | Type | Description
+
+| `override-with`
+| N
+| String
+| The orb job name to override the configuration with. (Both URL-based and registry orbs are supported)
```

Comment:
Is there anything public we can link to yet that states what a URL orb is? I think probably not but worth a check

Reply:
Not that I could find :/